spec/helpers.rb

```diff
@@ -0,0 +1,5 @@
+module Helpers
+  def card_new (orig, transl) #new card without .save
```

Comment:
Missing space after #.<br>Do not put a space between a method name and the opening parenthesis.